openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -689,11 +742,11 @@ class CourseOverviewImageSet(TimeStampedModel):
     large_url = models.TextField(blank=True, default="")
 
     @classmethod
-    def create_for_course(cls, course_overview, course=None):
+    def create_or_update_for_course(cls, course_overview, course=None):
```

Comment:
Consider changing this to simply `create_or_update`.  The `for_course` part isn't really needed and adds confusion when we contrast it with the `create_or_update_from_course` above.